Makefile

```diff
@@ -227,15 +230,12 @@ parser/cparser:
 	@echo "Building the C parser..."
 	@$(MAKE) -C parser
 
-clang-tools/call-sites: clang-tools/clang-kast
-
-.PHONY: clang-tools/clang-kast
-clang-tools/clang-kast: clang-tools/Makefile
-	@echo "Building the C++ parser..."
-	@$(MAKE) -C clang-tools
+$(CLANG_TOOLS_BIN)/%: $(CLANG_TOOLS_BUILD_DIR)/Makefile
+	@$(MAKE) -C $(CLANG_TOOLS_BUILD_DIR) $*
 
-clang-tools/Makefile:
-	@cd clang-tools && cmake .
+$(CLANG_TOOLS_BUILD_DIR)/Makefile:
```

Comment:
this needs to either be a phony target or depend on CMakeLists.txt.

Reply:
I can't believe I missed that. Thanks.